Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -201,6 +201,15 @@ struct PerWebProcessState {
 
 #endif // PLATFORM(IOS_FAMILY)
 
+#if HAVE(UIFINDINTERACTION)
+enum FindOverlayView {
```

Comment:
Make this an enum class please?

Reply:
Oh, I see why you did it like this. My other comment speaks in a different direction if you want to consider that.